diff --git a/sanoid b/sanoid
index ddd457f..d5d2a95 100755
--- a/sanoid
+++ b/sanoid
@@ -18,7 +18,7 @@ use Time::Local;      # to parse dates in reverse
 
 my %args = ("configdir" => "/etc/sanoid");
 GetOptions(\%args, "verbose", "debug", "cron", "readonly", "quiet",
-                   "monitor-health", "force-update", "configdir=s",
+                   "monitor-health", "force-update", "configdir=s", "defaults-configdir=s",
                    "monitor-snapshots", "take-snapshots", "prune-snapshots", "force-prune",
                    "monitor-capacity"
           ) or pod2usage(2);
@@ -34,8 +34,14 @@ my $pscmd = '/bin/ps';
 my $zfs = '/sbin/zfs';
 my $zpool = '/sbin/zpool';
 
+my $defaults_conf_dir;
+if (exists($args{'defaults-configdir'})) {
+  $defaults_conf_dir = "$args{'defaults-configdir'}";
+} else {
+  $defaults_conf_dir = "$args{'configdir'}";
+}
 my $conf_file = "$args{'configdir'}/sanoid.conf";
-my $default_conf_file = "$args{'configdir'}/sanoid.defaults.conf";
+my $default_conf_file = "$defaults_conf_dir/sanoid.defaults.conf";
 
 # parse config file
 my %config = init($conf_file,$default_conf_file);
@@ -1526,6 +1532,8 @@ Assumes --cron --verbose if no other arguments (other than configdir) are specif
 Options:
 
   --configdir=DIR       Specify a directory to find config file sanoid.conf
+  --defaults-configdir=DIR
+                        Specify a directory to find the sanoid.defaults.conf file
 
   --cron                Creates snapshots and purges expired snapshots
   --verbose             Prints out additional information during a sanoid run
